djarg/views.py

```diff
@@ -148,11 +148,11 @@ def get_objects(self):
 
         return objects
 
-    #: Get Queryset
-    get_queryset = edit_views.SingleObjectMixin.get_queryset
-    get_queryset.__doc__ = """
-        Uses Django's SingleObjectMixin get_queryset implementation.
-    """
+    def get_queryset(self):
+        if isinstance(self.queryset, arg.Lazy):
+            return arg.load(self.queryset, request=self.request)
+        else:
+            return edit_views.SingleObjectMixin.get_queryset(self)
```

Comment:
I can't call `super().get_queryset()` because the `MultipleObjectsMixin()` class, which is being defined here, doesn't derive from a Django generic view, like the `SingleObjectMixin()` class does above.
There are probably some alternative ways to do this part. Perhaps we should actually add a Django generic view into the hierarchy - but I figured this seemed like the most straight-forward way.

I'm still not too comfortable doing larger changes, as I'm still getting familiar with the framework. But I'd gladly take suggestions and investigate this further.

Reply:
Thinking about this a bit deeper, I kinda have 2 thoughts:

Should we use the `MultipleObjectMixin.get_queryset()`  from `https://github.com/django/django/blob/master/django/views/generic/list.py`?

Or - should we go further and make the multi-object view derive from that `MultipleObjectMixin` class?

Reply:
Tho.. that mixin brings in a lot of other things we _may_ not need - as pagination etc. Which I'm not sure how well plays with this library.